sql/pg_net--0.1.sql

```diff
@@ -101,6 +110,53 @@ begin
 end
 $$;
 
+-- Interface to make an async request
+-- API: Public
+create or replace function net.http_post(
+    -- url for the request
+    url text,
+    -- ContentType of the POST request
+    content_type text,
```

Comment:
Since we have the `headers` parameter this can be removed. It'd be a 
```json
...
headers := '{"Content-Type": "text/html; charset=UTF-8"}'
...
```